mmengine/visualization/vis_backend.py

```diff
@@ -417,7 +417,12 @@ def add_config(self, config: Config, **kwargs) -> None:
         Args:
             config (Config): The Config object
         """
-        self._wandb.config.update(dict(config))
+        if 'allow_val_change' in self._init_kwargs:
+            self.allow_val_change = self._init_kwargs['allow_val_change']
+        else: 
+            self.allow_val_change = False
+        self._wandb.config.update(
+            dict(config), allow_val_change=self.allow_val_change)
```

Comment:
If `allow_val_change` exists in `self._init_kwargs`, it should be passed in `wandb` when calling `wandb.init(**self._init_kwargs)` in `_init_env` method.

Reply:
![image](https://user-images.githubusercontent.com/62554593/223006605-2b4b4dfc-28fd-4f3b-851d-f9e1345fce97.png)
I have tried to set [resume=True/"allow", id="xxx", allow_val_change=True] in wandb.init, but still report this error。
But After I set the allow_val_change=True in wandb.config.update(), it stopped prompting this error and successfully ran resume

Reply:
Hi @zgzhengSEU , thanks for your contribution. Could you also open an issue in wandb to report this issue?